Automattic/wp-calypso
# Fix theme showcase button overlapping.

See issue #5803 that this PR addresses.

Header Cake has the second button `spacer` disabled and mobed to back. 
This button is not used/visible in Theme Sheet and when layout changes
from two pane to one pane it brakes `Pick this design` button functionality
by overlapping it. 

To test: 

1. Open Theme Sheet.
2. Reduce width of browser window to the point when layout changes from two panes to one pane
3. Inspect `Pick this design` button to check that it links to only one place

Test live: https://calypso.live/?branch=fix/5803-theme-showcase-button-behavior
